Offseason-Remade/src/main/java/frc/robot/RobotMap.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot;
+
+/**
+ * The RobotMap is a mapping from the ports sensors and actuators are wired into
+ * to a variable name. This provides flexibility changing wiring, makes checking
+ * the wiring easier and significantly reduces the number of magic numbers
+ * floating around.
+ */
+public class RobotMap {
+
+  // Device ID's for devices connected to the CAN bus
+  public static class CAN {
+    // drivetrain
+    public static final int DRIVETRAIN_RIGHT_FRONT_MOTOR = 9;
+    public static final int DRIVETRAIN_RIGHT_MIDDLE_MOTOR = 8;
+    public static final int DRIVETRAIN_RIGHT_REAR_MOTOR = 7;
+    public static final int DRIVETRAIN_LEFT_FRONT_MOTOR = 5;
+    public static final int DRIVETRAIN_LEFT_MIDDLE_MOTOR = 4;
+    public static final int DRIVETRAIN_LEFT_REAR_MOTOR = 3;
+
+    public static final int DRIVETRAIN_RIGHT_ENCODER = 12;
+
+    // lift
+    public static final int LIFT_RIGHT_MOTOR = 2;
+    public static final int LIFT_LEFT_MOTOR = 6;
+
+    // cargo collector
+    public static final int CARGO_HOLDER_MOTOR = 11;
```

Comment:
THAN WHY THE **_COMMANT_** IS 
// cargo collector
WHY MOSHE WHYYYYYYYYYYYYYY???